services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(
+            show_time__date=datetime.strptime(session_date, "%Y-%m-%d").date())

```

Comment:
```show_time__date=session_date```
will work in the same way

Reply:
Yes, but I learned about it only in the next episode of "Danilo and Django"  :)

Reply:
I was thinking that if the string with the date is not formatted as in the table, should we reformat it or does Django format the dates?

Reply:
Yes. You are totally right!